lib/octokit/client/issues.rb

```diff
@@ -329,6 +329,19 @@ def issue_timeline(repo, number, options = {})
       def add_assignees(repo, number, assignees, options = {})
         post "#{Repository.path repo}/issues/#{number}/assignees", options.merge({:assignees => assignees})
       end
+
+      # Remove assignees from an issue
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param number [Integer] Issue number
+      # @param assignees [Array] Assignees to be removed
+      # @return [Sawyer::Resource] Issue
```

Comment:
We're missing the options hash in the documentation here.

Reply:
Thanks @kytrinyx for the review.

What are you expecting here? The `options` argument doesn't seem very useful outside of consistency efforts with the rest of this library, as [the GitHub API](https://developer.github.com/v3/issues/assignees/#remove-assignees-from-an-issue) doesn't allow for additional parameters.

Reply:
Unless I'm missing something, the options hash is where you would pass things like custom accept headers.

Reply:
Added.

Reply:
Appreciate it! 🌷

Reply:
Dang. I was a bit quick to merge. Still not seeing the options hash—I'll add it in a separate PR.